mobile/cmp/popover/Popover.js

```diff
@@ -0,0 +1,191 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2021 Extremely Heavy Industries Inc.
+ */
+import {hoistCmp, HoistModel, useLocalModel} from '@xh/hoist/core';
+import {observable, action, makeObservable} from '@xh/hoist/mobx';
+import {div} from '@xh/hoist/cmp/layout';
+import {elementFromContent, createObservableRef} from '@xh/hoist/utils/react';
+import {isNil, isFunction} from 'lodash';
+import {usePopper} from 'react-popper';
+import ReactDom from 'react-dom';
+import classNames from 'classnames';
+import PT from 'prop-types';
+
+import './Popover.scss';
+
+/**
+ * Popovers display floating content next to a target element.
+ *
+ * The API is based on a stripped-down version of Blueprint's Popover component
+ * that is used on Desktop. Popover is built on top of the Popper.js library.
+ *
+ * @see https://popper.js.org/
```

Comment:
Sorry to mention this late in the game, but did you catch that Blueprint is starting to phase in a new implementation, based on the latest popper library?  

They are currently bundling it as a separate package - https://blueprintjs.com/docs/#popover2-package.  I just saw this the other day and was thinking that we should be sure to review this - and likely us it - for a desktop dialog component.  But it might well apply here too in terms of taking the latest popper and seeing how BP has changed their implementation / approach.

Reply:
(to be clear - I don't think we need to open this all up and swap over - but we should look at some point soon, and I wanted to mention it in case we are struggling at all with popper on mobile and there is a chance the new version is significantly improved)

Reply:
we did look at the new popover for sure, and should move to it.  did not seem like a major change